packages/amplify-util-mock/package.json

```diff
@@ -66,7 +66,7 @@
     "@types/semver": "^7.1.0",
     "@types/which": "^1.3.2",
     "amplify-function-plugin-interface": "^1.9.4",
-    "amplify-nodejs-function-runtime-provider": "^2.2.28",
+    "amplify-nodejs-function-runtime-provider": "2.3.4-cdkv2.0",
```

Comment:
This will need caret version back when we release GA. https://app.asana.com/0/1202389680978480/1203201977550367/f